### ISSUE TITLE
Accessibility: jumps in review

### ISSUE THREAD

**Issue by codeofdusk** [microsoft/terminal#5143]
# Environment

Windows build number: [run `[Environment]::OSVersion` for powershell, or `ver` for cmd]: 1909 (18363.720)

Windows Terminal version (if applicable): Terminal as of 5bcf0fc.

Any other software? NVDA, Narrator.

# Steps to reproduce

1. type:

``` shell
echo 1
echo 2
echo 3
```

2. Using NVDA with Windows Terminal support or Narrator, review a few previous lines of output. You should see the prompt, then 3, 2, 1.
3. wait about three seconds.
4. Try to review previous line again.

# Expected behavior
The screen reader keeps its position, reporting "2" for example if the last-reviewed line was "3".

# Actual behavior
The screen reader reads the prompt again, then starts back over with 3, almost as if the cursor bounced back.

**Comment by codeofdusk** [comment]
Cc @carlos-zamora.

**Comment by codeofdusk** [comment]
This cursor bouncing does not occur in open console (that works as expected).

**Comment by carlos-zamora** [comment]
I tested this with NVDA and Narrator. NVDA 2020.1 does not reproduce the above example, but Narrator does.

After disabling cursor blinking, scan mode on Narrator is no longer interrupted. After #5135 goes in, I may be able to leverage the new winrt event to only fire a CursorChanged event on cursor movements, instead of every time we render the cursor. This should be a lot less noisy.

**Comment by codeofdusk** [comment]
> After #5135 goes in, I may be able to leverage the new winrt event to only fire a CursorChanged event on cursor movements, instead of every time we render the cursor. This should be a lot less noisy.

Now that #5135 has been merged, does this still seem possible?

**Comment by carlos-zamora** [comment]
> > After #5135 goes in, I may be able to leverage the new winrt event to only fire a CursorChanged event on cursor movements, instead of every time we render the cursor. This should be a lot less noisy.
> 
> Now that #5135 has been merged, does this still seem possible?

Yeah. Working on it now. Hoping to get a PR in by the end of the day or tomorrow 😊
